spec/modules.md

```diff
@@ -262,7 +264,7 @@ This algorithm takes a string `argument` and [configuration](#configuration)
 
   [executed]: spec.md#executing-a-file
 
-  * If `config` is not empty, throw an error.
+  * If `config` has a different ID than before and is not empty, throw an error.
```

Comment:
Currently, nothing in this spec says to create a config with anything other than a unique ID, so this is really just saying "if `config` is a different object than the previous one". If that's all we need to specify, then we can do away with the "opaque ID" concept entirely and just say that directly. But if we want to consider different configuration objects to be "the same"—for example if they're created with `@forward as` or `@forward with`—we need to explicitly say "create a new configuration with the same ID as ..." when they're created.

Reply:
I have added a note on `ForwardWithArgument` but it's a bit far so I added a note below, so info is not too far apart